### DISCUSSION TITLE
FragPipeAnalystR package tutorial/documentation

### DISCUSSION THREAD

**Discussion by MelissaVorster** [Nesvilab/FragPipe-Analyst-doc#33] [Q&A] [answered]
Good day Team,
I have 3 LFQ and 1 TMT dataset that I have run through FragPipe and now would like to interpret using FragPipeAnalyst, but would like to make amendments and have a bit more control on the generated plots like indicating the thresholds on the volcano plot etc. I tried to use the R-package but could not get the package to work. Is it still under construction or can you please provide me with the necessary help file for this package?

This would be much appreciated.
Melissa

**Comment by hsiaoyi0504** [comment, marked as answer]
@MelissaVorster Sorry for the delay. Now we have more tutorial toward the FragPipeAnalystR here: https://www.nesvilab.org/FragPipeAnalystR/. Please let us know if anything still doesn't work. Thanks.
